doc/rst/source/supplements/seis/seis_extra_cols.rst_

```diff
@@ -0,0 +1,16 @@
+Data Column Order
+-----------------
+
+The **-S** option determines how many data columns are required to generate
+the selected symbol.  In addition, your use of options **-I** and **-t** will
+require extra columns, as will a **-S** option without the *scale*.  The
+order of the data record is fixed, even if not all items may be activated.
+We expect data columns to come in the following order::
+
+    lon lat [depth] symbol-columns [scale] [intens] [transp [transp2]] [trailing-text]
+
+where *symbol-columns* represent the normally required data columns, and items
+given in brackets are optional and under the control of the stated options
+(the trailing text is always optional).  **Notes**: (1) *depth* is normally required
+but is not expected if **-Fo** is selected.  (2) You can use **-i** to rearrange your
+data record to match the expected format.
```

Comment:
I just checked **meca** and **coupe**. **meca** has the **-Fo** option, but **coupe** doesn't.

Reply:
I guess it would make no sense for coupe to not read depths - hard to make those crosssection plots.  OK, so updating note to be meca-specific.